lib/Kickbox/HttpClient/HttpClient.php

```diff
@@ -99,9 +99,13 @@ private function request($path, array $body = [], $httpMethod = 'GET', array $op
             unset($options['headers']);
         }
 
-        $options['body'] = json_encode($body);
+        if($httpMethod == 'PUT') {
+            $options['body'] = join("\n",$body);
```

Comment:
I agree and that was essentially just to get it to the point where it works well enough. I think if you really want to do it right the PUT request should accept a stream or an already formatted body string.